.dockerignore

```diff
@@ -0,0 +1 @@
+bin/
```

Comment:
This little icon on GitHub is nagging about wanting you to add another line to the end of this file, as is the typical Unix convention (newline at the end of every file).

Since I have the GitHub-given power, I'm just going to apply this change myself with the little pencil icon.